util/ptr.go

```diff
@@ -0,0 +1,3 @@
+package util
+
+func Boolptr(b bool) *bool { return &b }
```

Comment:
Please move this inside the server package as an unexported function.

Reply:
I found out that these helper functions already exist in `utils`. I'm going to use them instead.